src/content/docs/vulnerability-management/changing-vulnerability-status.mdx

```diff
@@ -0,0 +1,293 @@
+
+        Click the vulnerability that you wish to change to affected.
+
+        <img
+            title="Vulnerabilities tab scoped to a specific entity and only ignored instances"
+            alt="An image showing the scoped vulnerabilities tab of Vulnerability Management filtered to only ignored instances"
+            src={vmAffectedEntityFiltered}
+        />
+    </Step>
+    <Step>
+        #### Change to affected
+
+        Click **Change to affected** to begin the process of setting this vulnerability to affected. 
+        
+        Removing the ignored status will result in the vulnerability getting included in default views and counts on this entity. 
+        
+        Other instances of this vulnerability on other entities will still remain ignored.
+
+        <img
+            title="Vulnerability instance details affected button"
+             alt="An image showing the change to affected button on vulnerability instance details page"
+            src={vmAffectedEntityDetails}
+        />
+    </Step>
+    <Step>
+        #### Fill out form
+
+        Fill out the form explaining the reasoning behind changing this vulnerability to affected.
+        
+        <img
+            title="Vulnerability instance details form"
+            alt="An image showing the affected form on vulnerability instance details page"
+            src={vmAffectedEntityForm}
+        />
+    </Step>
+    <Step>
+        #### Save the change
+
+       Click **Save** when you are ready to submit the change of status. 
+       
+       Validate that the status has changed from ignored to affected. Notice that the changelog keeps a historical track of all modifications.
+    </Step>
+</Steps>
+
+### How to bulk remove ignored status (change to affected)
+
+<Steps>
+    <Step>
+        #### Navigate to NRVM
+
+        Go to [one.newrelic.com](https://one.newrelic.com/) > **All capabilities** > Select the **Vulnerability Management** tile > click the **Vulnerabilities** tab 
+    </Step>
+    <Step>
+        #### Find a vulnerability with ignored instances
+
+        You can find ignored instances by clicking the **Settings Icon** and clicking the **Ignored** checkbox to add an ignore column to your vulnerability table.
+
+        Now you can understand which vulnerabilies have instances ignored. Alternatively, you can change the status filter to **Ignored** to see vulnerabilities where all instances have been ignored.
+
+        Select a vulnerability where the ignored count is greater than 0. 
+
+        <img
+            title="Vulnerabilities tab with ignored column added"
+            alt="An image showing the all entities vulnerabilities tab with an ignore column added"
+            src={vmAffectedBulkVulnList}
+        />
+    </Step>
+    <Step>
+        #### Change the filter
+
+        Click the filter **Status = Affected** and change it to **Ignored**. Now only entities with the vulnerability marked as ignored will be visible. 
```

Comment:
What's the difference between this step and the step that adds the ignore column? When might someone choose one method over the other?